cgmanifest.json

```
@@ -0,0 +1,14 @@
+{
+    "Registrations": [
+        {
+            "Component": {
+                "Type": "other",
+                "Other": {
+                    "Name": "cmake module for adding code coverage",
+                    "Version": "any version",
+                    "DownloadUrl": "https://github.com/bilke/cmake-modules/blob/master/CodeCoverage.cmake"
+                }
+            }
+        }
+    ]
+}
```

VS auto-format suggestion

```suggestion
{
  "Registrations": [
    {
      "Component": {
        "Type": "other",
        "Other": {
          "Name": "cmake module for adding code coverage",
          "Version": "any version",
          "DownloadUrl": "https://github.com/bilke/cmake-modules/blob/master/CodeCoverage.cmake"
        }
      }
    }
  ]
}
```